src/index.js

```diff
@@ -1,10 +1,6 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+import { createRoot } from "react-dom/client";
+import App from "./App";
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+const container = document.getElementById("root");
+const root = createRoot(container);
+root.render(<App />);
```

Comment:
I applaud your removal of strict mode and do the same on some of my projects. Just make sure you understand the ramifications of this going forward. There is a fair chance we will talk about strict mode and how it affects some of the characteristics of useEffect.